internal/exec/utils.go

```diff
@@ -152,13 +152,14 @@ func processArgsConfigAndStacks(componentType string, cmd *cobra.Command, args [
 		return configAndStacksInfo, err
 	}
 
-	argsAndFlagsInfo, err := processArgsAndFlags(args)
+	argsAndFlagsInfo, err := processArgsAndFlags(componentType, args)
 	if err != nil {
 		return configAndStacksInfo, err
 	}
 
 	configAndStacksInfo.AdditionalArgsAndFlags = argsAndFlagsInfo.AdditionalArgsAndFlags
 	configAndStacksInfo.SubCommand = argsAndFlagsInfo.SubCommand
+	configAndStacksInfo.SubCommand2 = argsAndFlagsInfo.SubCommand2
```

Comment:
`atmos terraform workspace select`

in the code the subcommand is `workspace` and subcommand2 is `select`.
Terraform calls it <subcommand>, but we already have a subcommand since `atmos` works not only with terraform.

Reply:
is SubCommand2 simply the word following the SubCommand ?

Yes